packages/db/src/search/ExecutionSearch.ts

```diff
@@ -0,0 +1,150 @@
+import { Knex } from 'knex';
+import Logger from '@cumulus/logger';
+import { ApiExecutionRecord, ExecutionRecordStatus } from '@cumulus/types/api/executions';
+import { BaseSearch } from './BaseSearch';
+import { DbQueryParameters, QueryEvent } from '../types/search';
+import { translatePostgresExecutionToApiExecutionWithoutDbQuery } from '../translate/executions';
+import { PostgresExecutionRecord } from '../types/execution';
+import { TableNames } from '../tables';
+import { BaseRecord } from '../types/base';
+
+const log = new Logger({ sender: '@cumulus/db/ExecutionSearch' });
+
+interface ExecutionRecord extends BaseRecord, PostgresExecutionRecord {
+  cumulus_id: number,
+  arn: string,
+  async_operation_cumulus_id: number,
+  asyncOperationId: string,
```

Comment:
this breaks the pattern of snake_case, can it be changed or does it need to be camel case?